packages/cypress-commands/src/commands/fillAndSubmitLoginForm.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {Object} args
+ * @param {string} args.userName
+ * @param {string} args.password
+ * @param {string} [args.loginUrl]
+ * When not provided, the server field will be ignored.
+ * If the server is already set through an env var,
+ * then not providing this property will produce the expected result
+ *
+ * @returns {void}
+ */
+export const fillAndSubmitLoginForm = ({ loginUrl, userName, password }) => {
+    loginUrl && cy.get('#server').type(loginUrl)
+    cy.get('#j_username').type(userName)
+    cy.get('#j_password').type(password)
+    cy.get('{loginsubmit}', { prefix: 'dhis2-adapter' }).click()
+}
+
+Cypress.Commands.add('fillAndSubmitLoginForm', fillAndSubmitLoginForm)
```

Comment:
Hey, I know this is being added for the capture app use-case. Backstory time: sometime ago we had this bug that Gintare, Jeniffer and others solved in the `login` helper of the utils. Basically we couldnt login by posting a request to the server as a workaround we used the ui to login.. 

That said and if there isnt any other use cases for this utility perhaps you can think of removing it.

Reply:
I agree with @paschalidi: if `loginBasicAuth` works reliably, then I'd classify this command as redundant.

Also, if there are compelling reasons for keeping it, I think we should rename it. My qualm with the current name is that it doesn't include a reference to "platform app". I think it should do though, since this login form is specific to the app-shell.

Reply:
will be removed as it's an unnecessary abstraction. Apps can do it themselves